### ISSUE TITLE
feat: Allow to remove all cordova related code

### ISSUE THREAD

**Issue by IljaDaderko** [ionic-team/capacitor#2356]
# Feature Request

## Describe the Feature Request
Having started new project with capacitor I'm really excited about its future. I saw that a lot of cordova related stuff is still left in some file, we even have `capacitor.build.gradle` file that is generated with cordova dependency every time `capacitor update` is ran.

This feels like unnecessary / legacy stuff in new projects and if I am correct capacitor works perfectly fine without cordova?

Would it be possible to add section in docs or some flag to cli that allows to generate cordova free project / shows how to remove it from existing project?

## Platform Support Requested

- [x] Android
- [x] iOS
- [ ] Electron
- [ ] Web

## Describe Preferred Solution
1. Ability to un-select cordova as dependency when creating new project
2. Documentation for removing cordova in existing apps

## Describe Alternatives
Perhaps a blog post covering this?

## Related Code

These are all places where I found reference to cordova files.

- `/android/.gitignore`
- `/android/app/build.gradle`
- `/android/app/capacitor.build.gradle`
- `/android/app/src/main/res/xml/config.xml`
- `/android/settings.gradle`
- `/ios/.gitignore`
- `/ios/App/App/config.xml`
- `/ios/App?Podfile`

## Additional Context
Perhaps it is enough to just remove all cordova references from these files, but once again there are files like `capacitor.build.gradle` that are auto generated and reference it and I think project would break?

**Comment by jcesarmobile** [comment]
Issues tagged with feature request are closed but tracked for reactions to gauge interest.

**Comment by dotNetkow** [comment]
Docs on migrating, then removing Cordova [are here](https://capacitor.ionicframework.com/docs/cordova/migrating-from-cordova-to-capacitor).

**Comment by jcesarmobile** [comment]
No, this is about the cordova compatibly code that capacitor projects include.

**Comment by IljaDaderko** [comment]
Yep. For me personally I think only autogenerated `capacitor.build.gradle` is what's preventing me from completely removing all references to cordova. Perhaps additional flag in capacitor config file like `cordova: false` (default true) would be enough? However, I am not sure if capacitor relies on some cordova files to exist during builds.

**Comment by harveyea30** [comment]
This one is in progress.

**Comment by markemer** [comment]
This has been merged in #8328, into the [next](https://github.com/ionic-team/capacitor/tree/next) branch for Capacitor 9.
